Lib/defconQt/glyphView.py

```diff
@@ -1472,7 +1833,7 @@ def addHorizontalMetrics(self):
             ("Baseline", 0),
             ("x-height", font.info.xHeight),
             ("Cap height", font.info.capHeight),
-            ("Ascender", font.info.ascender),
+            ("Ascender", font.info.ascender)
```

Comment:
Yeah, NP. In JavaScript it's illegal, and I'm writing a lot of that ...
I use a lot of comma first style for the same reason though.